web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -90,6 +97,45 @@
 
   let showAssetPath = false;
   const toggleAssetPath = () => (showAssetPath = !showAssetPath);
+
+  let isShowChangeDate = false;
+
+  async function handleConfirmChangeDate(event: CustomEvent<string>) {
+    isShowChangeDate = false;
+    if (asset.exifInfo) {
+      asset.exifInfo.dateTimeOriginal = event.detail;
+    }
+    try {
+      await api.assetApi.updateAsset({
+        id: asset.id,
+        updateAssetDto: {
+          creationDate: event.detail,
+        },
+      });
+      notificationController.show({ message: 'Metadata updated please reload to apply', type: NotificationType.Info });
```

Comment:
Can we just change the `asset` var and let svelte handle this? (cc @jrasm91 / @alextran1502 - maybe not because that var is handed down from the timeline?)

Reply:
I don't remember what used to be here, but I think it's need fixed now?

Reply:
My intent was rather than asking the user to reload the page, can we just update it in place?